platform/administer/templates/overview.mdx

```diff
@@ -0,0 +1,820 @@
+resource "kubernetes_manifest" "environment_templates" {
+  for_each = local.templates
+  
+  manifest = {
+    apiVersion = "vcluster.loft.sh/v1"
+    kind       = "VClusterTemplate"
+    
+    metadata = {
+      name      = "${each.key}-template"
+      namespace = "vcluster-templates"
+    }
+    
+    spec = {
+      displayName = "${title(each.key)} Environment Template"
+      description = "Template optimized for ${each.key} workloads"
+      
+      template = {
+        spec = {
+          controlPlane = {
+            replicas = each.value.replicas
+            resources = {
+              limits = {
+                cpu    = each.value.cpu_limit
+                memory = each.value.memory_limit
+              }
+            }
+          }
+          
+          storage = {
+            persistence = each.value.persistence
+            size        = each.value.storage_size
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+## Template versioning capabilities
+
+**Note: template versioning requires Loft Pro**
+
+Template versioning provides a way to manage template evolution over time. When you have versioning enabled, each change to a template creates a new version that you can track, reference, and rollback to if needed.
+
+### How versioning works
+
+Versioning operates automatically once you enable it:
+
+1. **Initial creation**: When you first create a template, it exists without an explicit version number
+2. **First update**: The moment you modify the template (through any method), the system creates version 1.0.0 
+3. **Subsequent updates**: Each additional change increments the version number following semantic versioning principles
+4. **Version tracking**: The system maintains a complete history of all template versions with their configurations and metadata
+
+### Version management operations
+
+**List all versions of a template:**
+
+```bash
+# Show version history for a specific template
+loft list templates development-template --show-versions
+
+# Output example:
+# NAME                  VERSION   CREATED             MODIFIED
+# development-template  1.2.3     2024-01-15 10:30    2024-02-20 14:45
+# development-template  1.2.2     2024-01-10 09:15    2024-01-15 10:30
+# development-template  1.2.1     2024-01-05 16:20    2024-01-10 09:15
+```
+
+**Deploy a vCluster using a specific template version:**
+
+```bash
+# Create vCluster with latest template version (default behavior)
+loft create vcluster my-development-cluster --template development-template
+
+# Create vCluster with specific template version
+loft create vcluster my-stable-cluster \
+  --template development-template \
+  --template-version 1.2.2
+```
+
+**Compare template versions:**
+
+```bash
+# View differences between template versions
+loft diff template development-template --from-version 1.2.2 --to-version 1.2.3
+```
+
+### Version rollback scenarios
+
+Template versioning becomes crucial when you need to rollback problematic changes:
```

Comment:
This section might be confusing. I've added corrections to the commands, but they would only apply to each vCluster. If the goal is to modify multiple vCluster to use a new template version, that's only possible using "wild card" versions. If a VirtualClusterInstance refers to a template version with an `x` (ex: `1.0.x`, it will automatically upgrade when a new version (like `1.0.1`) is created.